test/pylib/schema_helper.py

```diff
@@ -0,0 +1,299 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+# This file provides helpers for table creation and schema changes.
+"""Provides helpers for CQL table management.
+
+Classes:
+    Tables
+    TestTables
+"""
+
+
+from __future__ import annotations
+from abc import ABCMeta
+import asyncio
+import itertools
+import random
+import uuid
+from typing import Type, List, Set, Union, TYPE_CHECKING
+if TYPE_CHECKING:
+    from cassandra.cluster import Session as CassandraSession            # type: ignore
+
+
+new_keyspace_id = itertools.count(start=1).__next__
```

Comment:
Let's use unique_name()?

Reply:
I don't like `unique_name()` as it's ugly, hard to repro (takes from timestamp), and hacky.

An itertools generator is easier to read, easier to repro, and even thread safe.

Reply:
Fine, let's put your logic into unique_name() then. My only point is that we don't have two ways of making names. We could make the names randomness depends on a single rnd seed, too.

Reply:
Removed this to use the keyspace fixture